### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by mkatychev**
* Moved `docs/tuto.md` to `docs/tutorial.md` hopefully this will fix the borken _Content_ footer in 
https://nickel-lang.org/user-manual/introduction to hopefully mirror the TOC here: https://github.com/tweag/nickel-lang.org/blob/1f309ec8868a83d15bb084037ea266f81c8ab2ea/gatsby-config.js#L25
* Fixed invalid command

### CODE REVIEW

File: doc/manual/introduction.md
Comment (mkatychev) [inline doc/manual/introduction.md:96]: this generates https://nickel-lang.org/user-manual/tuto which is an invalid link.

File: doc/manual/contracts.md
Comment (yannham) [inline doc/manual/contracts.md:41]: Hmm, while your suggestion is equally valid, I don't think the original text is incorrect (or less correct) either. I think it's supposed to be a sentence in plain English, using the common word "number" on purpose.

While the difference is small, IMHO it's a bit less of a circular definition to say that

> `Number` check that something is a number

rather than

> `Number` checks that something is a `Number`

Typically, the contract was `Num` before 1.0, in which case the second wording would a bit more obscure.

```suggestion
2. Check that the result is a number.
```

**Review by yannham — CHANGES_REQUESTED** [review]
Thanks for contributing! I'm not sure about the documentation change, but the tutorial renaming is surely better. I've changed the target branch to `stable`, such that it's picked up by the website. Note for maintainer: once merged, we'll have to cherry-pick this on master as well [edit: couldn't do that because this commit starts from current master which is already quite ahead of stable, so let's do the converse: cherry-pick this commit on `stable` after merging]

File: doc/manual/contracts.md
Comment (mkatychev) [reply doc/manual/contracts.md:41]: makes sense

**Review by yannham — APPROVED** [review]

File: doc/manual/tutorial.md
Comment (mkatychev) [inline doc/manual/tutorial.md:125]: @yannham  One change, I forgot to push, `--export` is not a valid flag.

Comment (yannham) [reply doc/manual/tutorial.md:125]: Ah, good catch! Thanks

**Pull request merged**
